react-regex.js

```diff
@@ -174,5 +174,15 @@ ReactRegex = [
   {
     regex: /\stabindex=/g,
     replace: " tabIndex="
+  },
+  // Template.contentBlock
+  {
+    regex: /({{>\s*Template.contentBlock\s*}})/g,
+    replace: "{this.props.children}"
+  },
+  // @index -- match @index by itself {{@index}} or as a param to helper {{helper @index anotherParam}}
+  {
+    regex: /({{.*@index.*}})/g,
+    replace: "{index}"
```

Comment:
Wow I didn't notice they talked about it. This is supercool! Yeah, we need to spread this project even more, but we need to become fully Blaze 1 compatible first?